build/three.js

```diff
@@ -37318,7 +37318,7 @@
 
 				Cache.add( url, this );
 
-				if ( onLoad ) { onLoad( this ); }
+				if ( onLoad ) { onLoad( this || image ); }
```

Comment:
Please do not include build files into the PR.

Reply:
ok.